cmd/kubernetes/kubernetes_update.go

```diff
@@ -0,0 +1,72 @@
+package kubernetes
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/common"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var firewall string
+
+var kubernetesUpdateCmd = &cobra.Command{
+	Use:     "update",
+	Aliases: []string{"edit"},
+	Short:   "Update a Kubernetes cluster",
```

Comment:
You may want to be more specific here  "Update the configuration of your Civo Kubernetes cluster" or something like that --> This makes it more specific to the user that they are updating just the configuration platform side, if that makes sense.

Reply:
updated